### DISCUSSION TITLE
Traffic levels with few exporters

### DISCUSSION THREAD

**Discussion by Intervator** [akvorado/akvorado#1266] [Q&A]
Hello,
I have two routers, each one has its own BGP uplinks. Also, there is a link between these routers with iBGP. The problem is that when I filter graphs with OutIfDescription data is collected only from the same exporter as the downstream interface. The traffic graph is correct with sampling between routers' interfaces but incorrect without it. The other way is when I filter graphs by the customer DstAS. Both exporters collect data but traffic almost doubles with active sampling between routers and is correct without it. Is there any way to make it display correct traffic with both OutIfDescription or DstAS filter?
![image](https://github.com/akvorado/akvorado/assets/36364560/8b535825-fe18-401c-860d-ff4873d21e1e)
![image](https://github.com/akvorado/akvorado/assets/36364560/5a74c87b-9dba-4abf-890d-441924f04023)

**Comment by vincentbernat** [comment]
Either:
- collect traffic on internet-facing interfaces on ingress and egress
- collect traffic on all interfaces on ingress and classify them correctly

**Reply by Intervator** [reply]
Thank you for your answer. Sampling is active only on our upstream and peering sessions of both routers. Is there any way to see all outgoing traffic to our customers, using OutIfDescription filter and SrcAS dimension? Now it is showing only traffic coming from uplinks of the same exporter as the customer's interface and ignoring traffic going to the customer from uplinks of another router through iBGP. Traffic levels are correct only if I'm using customers' DstAS as filter. Then data is collected from both exporters.
